ccloud/beginner-cloud/start.sh

```diff
@@ -15,11 +15,11 @@ source ../../utils/ccloud_library.sh
 
 ccloud::validate_version_ccloud_cli 1.7.0 || exit 1
 ccloud::validate_logged_in_ccloud_cli || exit 1
+ccloud::prompt_continue_ccloud_demo || exit 1
```

Comment:
@awalther28 approve, LGTM.  

One more note, if you're willing, I think we can remove `check_expect || exit 1` (line 21 below) but no worries if you want to leave that for another PR.

Reply:
@ybyzek What does `check_expect` do?

Reply:
nvm

Reply:
Removed, looks like we don't use `expect` anymore in the `start.sh` script.

Reply:
Before Confluent Cloud CLI supported `~/.netrc` for saved logins, the demos used `expect` to login.